pubspec.yaml

```diff
@@ -90,3 +90,4 @@ flutter:
   # see https://flutter.dev/custom-fonts/#from-packages
   assets:
     - movie_images/
+    - movie_data/
```

Comment:
Can we changed this folder name to mock_data, because you are getting movie.json but also genres.json, not only movie data

Reply:
fixed